Improve wording in `Gameplay/Game modifier/Accuracy Challenge` (#14736)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Accuracy_Challenge/en.md

- **Minimum accuracy** (60%-99%, default: 90%): Trigger a failure if accuracy goes below this value.
- **Accuracy mode** (default: Maximum achievable): The mode of accuracy that will trigger failure.

>   - **Maximum achievable**: Fail if the player is not able to achieve this threshold until the [beatmap](/wiki/Beatmap) ends even when hitting perfectly.

Reviewer: maybe this is more straightforward? I don't understand what difference does "even when hitting perfectly" make here
```suggestion
  - **Maximum achievable**: Fail if the player has not achieved the threshold when the [beatmap](/wiki/Beatmap) ends.
```

Author: `when the beatmap ends` could be interpreted in my eyes as if the fail screen only appears at the actual end of the beatmap, not when the play counts as failed.

`is not able to achieve the threshold until the beatmap ends` is much more clear to me.

`even when hitting perfectly` can be omitted if you prefer that.

Reviewer: so the fail is immediate? how about... "Fail immediately if the threshold cannot be achieved by the end of the beatmap"?

Author: That's good yeah

>   - **Maximum achievable**: Fail if the player is not able to achieve this threshold until the [beatmap](/wiki/Beatmap) ends even when hitting perfectly.

Author: ```suggestion
  - **Maximum achievable**: Fail immediately if the threshold cannot be achieved by the end of the [beatmap](/wiki/Beatmap).
```

  - **Standard**: Fail if the accuracy goes strictly below the value.
- Restart on fail (default: disabled): Restart the map when the accuracy goes below the minimum value specified.

> *The mod is eligible for performance points regardless of the customisation options.*

Reviewer: ```suggestion
*The mod is rankable regardless of the customisation options.*
```
